Project.toml

```diff
@@ -0,0 +1,15 @@
+name = "PyCall"
+uuid = "438e738f-606a-5dbb-bf0a-cddfbfd45ab0"
+authors = ["Steven G. Johnson <stevenj@mit.edu>", "Yichao Yu <yyc1992@gmail.com>", "Takafumi Arakaki <aka.tkf@gmail.com>", "Simon Kornblith <simon@simonster.com>", "Páll Haraldsson <Pall.Haraldsson@gmail.com>", "Jon Malmaud <malmaud@gmail.com>", "Jake Bolewski <jakebolewski@gmail.com>", "Keno Fischer <keno@alumni.harvard.edu>", "Joel Mason <jobba1@hotmail.com>", "Jameson Nash <vtjnash@gmail.com>", "The JuliaPy development team"]
+version = "v1.91.2-DEV"
+
+[deps]
+Conda = "8f4d0f93-b110-5947-807f-2305c1781a2d"
+MacroTools = "1914dd2f-81c6-5fcd-8719-6d5c9610ff09"
+VersionParsing = "81def892-9a0e-5fdd-b105-ffc91e053289"
+
+[compat]
+Conda = "1.0"
+MacroTools = "≥ 0.4"
```

Comment:
It seems MacroTools 0.4 worked with Julia 0.7 nightly https://travis-ci.org/MikeInnes/MacroTools.jl/builds/288579177. So if we say `julia = "0.7, 1.0"` I think it makes sense to allow MacroTools 0.4.  Alternatively, we can do `MacroTools = "0.4, 0.5"` but it then makes this version of PyCall incompatible with MacroTools 0.6 and above.